src/riak_kv.proto

```diff
@@ -406,11 +403,7 @@ message TsRow {
 message TsCell {
   optional bytes binary_value = 1;
   optional sint64 integer_value = 2;
```

Comment:
Almost missed this, we should call this sint64_value instead of integer_value. We may add int32, etc in the future, and plain "integer" is usually reserved for the 32-bit ones.